llarp/path/pathset.cpp

```diff
@@ -156,7 +156,7 @@ namespace llarp
           {
             if (chosen == nullptr)
               chosen = itr->second;
-            else if (chosen->intro.latency > itr->second->intro.latency)
+            else if (chosen->intro.latency > 0s and chosen->intro.latency > itr->second->intro.latency)
```

Comment:
very tiny nitpick, but it feels like it should be `!= 0s` instead of `> 0s`, to better convey that what we care about is that it is nonzero.  Obviously any nonzero in this case is also positive, so the conditions are equivalent.

Reply:
good point